README.md

```diff
@@ -185,25 +170,21 @@ channels:
           $ref: "london.hammersmith.transport._public.tube.passenger.avsc"
 ```
 
-This spec will create 3 topics by prepending the id of the app to the 'owned' topics (channels that use a relative path and start with `_private`, `_protected` or `_public`):
+In SpecMesh terms - this file, and what it represents, is considered to be:
+- a data product (Data Mesh terminology)
```

Comment:
I actually disagree here.  I think a spec can define multiple data products.  Each topic can be a data product, not the spec as a whole.

Reply:
Make this "one or more data products" ?